trusted-committer-article.md

```diff
@@ -0,0 +1,296 @@
+      board with it 
+    - Commmunicate technical and community risks to responsible managers
+    - Help potential community members with them beeing empowered to make
+      contributions by talking to their managers
+    - Ensure there are users and potential contributors by promoting and
+      advertising the community and its goals
+    - Make sure there are opportunities for people to get a together regularly
+    - It is important to both give and take in the community
+      
+## Reducing Barriers To Entry
+
+- Soliciting contributions is one of the things that is more challenging with
+  InnerSource compared to Open Source. There are a number of reasons for that 
+    - The sheer number of potential contributors is lower in InnerSource
+    - Contributors will want to contribute during their work time. That means
+      they are more time constrained compared to doing Open Source after office
+      hours. Time spent on InnerSource might be time lost on reaching their
+      original goals.
+    - Work in InnerSource might not necessarily be part of the official 
+      performance goals of contributors.
+- It is therefore super important to make the process for making contributions
+  and for onboarding contributors as frictionless as possible to avoid wasting
+  a contributors time. 
+- There are a number of things a TC can do to achieve that state
+    - have a good readme.md, providing detailed instructions on how to get,
+      build, test and use the SW developed by the InnerSource community.
+    - have a good contribute.md which outlines what is expected of the
+      contributor to make a contribution. It should answer common questions
+      like
+        - Who do I contact in case of questions and how to reach her?
+        - How do I submit a bug report or feature request?
+        - What is the definition of done for a contribution; code, tests, docs?
+        - What are conventions for code style, branching or committ messages,
+          e. g.?
+        - What is expected of me in terms of supporting contributed code after
+          the contribution was accepted?
+        - What are the process steps that govern contributions?
+        - Is there a code of conduct or other guidelines to how the community
+          operates?
+    - Make it easy to run and test the SW locally
+- As a TC, I have to make sure there is someone in the community who is
+  available to answer questions of users and potential contributors.
+- Also make it clear that non-code contributions are welcome, such as writing
+  documentation, organizing events or creating artwork.
+- In addition to these "passive" means to facilitate contributions, a TC also
+  actively helps with that
+    - Publishing a list of "Newbie Tasks" and actively solicit contributions
+      rather than waiting for them to happen.
+    - TCs are expected to actively help new contributors to finish their PRs
```

Comment:
May want to explain what "actively" means.  Wouldn't want to scare away potential TCs with the idea that they need to finish someone else's half-baked work.